draft-ietf-rats-eat.md

```diff
@@ -71,18 +71,20 @@ contributor:
 normative:
   RFC2119:
   RFC7515:
-  RFC8949:
+  RFC8949: cbor
   RFC7252:
   RFC7519:
   RFC8126:
   RFC8174:
   RFC8259:
   RFC8392:
+  RFC8610: cddl
   RFC8610:
```

Comment:
this line should be removed.  it got there through a subsequent merge / rebase IIUC